modules/monitoring-common-terms.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-overview.adoc
+
+:_content-type: REFERENCE
+[id="openshift-monitoring-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+Alertmanager::
+The Alertmanager service handles alerts received from Prometheus. Alertmanager is also responsible for sending the alerts to external notification systems.
```

Comment:
The Alertmanager service handles alerts received from Prometheus
->
The Alertmanager handles alerts received from Prometheus

Reply:
Addressed